README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are empowered to call out and attempt to resolve disrespectful behavior, and admins are empowered to enforce the Code of Conduct.
+
+## The Short Version
+
+Be respectful of other people; respectfully call out anyone behaving inappropriately, ask them to stop, and direct them to this Code of Conduct; and if you need help resolving an issue, contact the admins. If you are unable to follow our Community Code of Conduct you will be removed from HangOps.
+
+## The Long Version
+
+### Respect
+
+HangOps is a community of people who care about running production systems reliably. We have diverse backgrounds, skills, talents, experiences, cultures, and opinions. We are an inclusive environment and we treat all individuals respectfully.
+
+We value respectful behavior above individual opinions.
+
+Respectful behavior includes:
+
+* Be considerate, kind, constructive, and helpful.
+* Avoid demeaning, discriminatory, harassing, hateful, or physically threatening behavior, speech, and imagery.
+* If you’re not sure, ask someone instead of assuming. No, really. Just ask the admins.
+* Model respectful behaviour, but also address disrespect when you see it. One of the best ways to create a respectful community is to speak up when you see disrespectful behavior—and not just rely on admins to act!
+
+### Resolve Peacefully
+
+If you see someone behaving disrespectfully, we urge you to respectfully dissuade them from such behavior.
+
+If you experience disrespectful behavior toward yourself or anyone else and feel in any way unable or unwilling to respond or resolve it respectfully (for any reason), please immediately bring it to the attention of the admins who will listen and work to resolve the matter.
+
+### Apologize for Mistakes
+
+Should you catch yourself behaving disrespectfully, or if someone calls you out on disrespectful behavior, please listen intently, own up to your words and actions, and apologize accordingly. No one is perfect, and even well-intentioned people make mistakes. What matters is how you handle them and that you avoid repeating them.
+
+### Consequences
+
+If the admins determine that someone is behaving disrespectfully, the admins may take any action they deem appropriate within HangOps, up to and including expulsion and exclusion from HangOps.
```

Comment:
`Harmful` is a better choice, from those

Respect has a lot of cultural ties, which do not always cross borders, and communicating it can be tricky.